### PR TITLE
Fully automate dev setup with Gitpod

### PR DESCRIPTION
**Pull request by ivelin**
This commit implements a fully-automated development setup using Gitpod.io, an
online IDE for GitHub and GitLab that enables Dev-Environments-As-Code.
This makes it easy for anyone to get a ready-to-code workspace for any branch,
issue or pull request almost instantly with a single click.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+tasks:
+  - init: npm install && npm run build
```
Comment (svenefftinge) [inline .gitpod.yml:2]: You could add a command phase, which gets execute after init on when you start a workspace.
```
    command: npm run serve
```

**Comment by geekyahmed** [comment]
Nice feature though

File: .gitpod.yml (hunk 1 above)
Comment (ivelin) [reply .gitpod.yml:2]: @svenefftinge thanks for the hint. I will add it and see how it works.

I am still learning how gitpod.yml works. Currently I'm able to build and run my node app from the command line terminal in gitpod. See screenshot.

Still not quite clear how the gitpod.yml init and command lines are integrated in the UX. 

[This document](https://www.gitpod.io/docs/config-start-tasks/) is helpful. A few screenshots or videos with example workflows would be great.

Comment (ivelin) [reply .gitpod.yml:2]: Screen Shot 2020-11-25 at 6 19 50 PM

Comment (ivelin) [reply .gitpod.yml:2]: I have a somewhat related question about github access. I opened this issue about it: https://github.com/gitpod-io/gitpod/issues/2287

Comment (svenefftinge) [reply .gitpod.yml:2]: I have no idea why you see that message, although you can push via terminal. Maybe you can add some additional information to the ticket, that would help us to reproduce the issue.

**Pull request merged**